shapeshift/web
# fix: use active swapper quote state

## Description

Updates swapper logic to use the trade quote for the active swapper (currently the best swapper, but soon to be "selected" swapper) at all times.

Note, this does not yet fix https://github.com/shapeshift/web/issues/4025 - coming soon in another PR.

## Notice


## Pull Request Type


## Issue (if applicable)

Closes https://github.com/shapeshift/web/issues/4037

## Risk

Medium, touches swapper quote logic.

## Testing

- Confirm that the bugs in the ticket noted above are fixed by following the replication steps
- Confirm the swapper still works as expected (if the quotes and gas look right it's probably fine - no tricky account stuff here)

### Engineering

☝️ 

### Operations

☝️ 

## Screenshots (if applicable)

N/A
